tmpl/wiki/weekly/2015-02-11.md

```diff
@@ -0,0 +1,153 @@
+11th Feb 2015: Improving quality, error handling, TLS merge and GSoC
+
+### Agenda ###
+
+- Improving Quality
+- Error Handling RFC
+- TLS merge progress
+- OPAM 1.2 only
+- GSoC Projects
+- FOSDEM demo
+- Bitcoin Piñata
+
+
+Attendees: Amir Chaudhry, David Kaloper, Thomas Leonard, Jon Ludlam,
```

Comment:
Done (and sorry!). Next time we don't leap in until I've done a proper roll call. :)